src/qml/README.md

```diff
@@ -76,6 +76,9 @@ they are not needed for static builds:
 ```
 sudo apt install qml-module-qtquick2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-layouts qml-module-qtquick-window2 qml-module-qt-labs-settings
 ```
+##### Important:
+
+In cases where the necessary dependencies cannot be installed directly on your system, you can resort to performing a `depends build`, ensuring compatibility across different distributions. Detailed instructions can be found in the [dependencies guide](https://github.com/bitcoin-core/gui-qml/blob/main/depends/README.md).
```

Comment:
What about something like this. Cuts to the point.
```suggestion
If you're unable to install the dependencies through your system's package manager, you can instead perform a [depends build](https://github.com/bitcoin-core/gui-qml/blob/main/depends/README.md).
```

Reply:
Done, thanks.